src/main/resources/avro/biodata.avdl

```diff
@@ -0,0 +1,133 @@
+@namespace("org.ga4gh.models")
+
+/**
+This protocol defines the "biodata" objects, which can be considered data
+representations of biological correlates.
+*/
+
+protocol Biodata {
```

Comment:
Now you have me confused:
- sequencing datasets are not a biological object
- don't they have the read api for this?

I persist that the prefix Bio is purely decorative, and naming an API
'data' is up there in coding sins along with one letter variable names.
On 14 Apr 2016 08:03, "Mark Diekhans" <notifications@github.com> wrote:

> In src/main/resources/avro/biodata.avdl
> <https://github.com/ga4gh/schemas/pull/606#discussion_r59670763>:
>
> > @@ -0,0 +1,133 @@
> > +@namespace("org.ga4gh.models")
> > +
> > +/**
> > +This protocol defines the "biodata" objects, which can be considered data
> > +representations of biological correlates.
> > +*/
> > +
> > +protocol Biodata {
>
> Daniel Zerbino <notifications@github.com> writes:
> Still, that doesn't make Biodata a good name. May I suggest Specimen,
> which describes both individuals and samples?
> This is intended to contain the metadata for all of the biological stuff
> that exists outside of the computer. It will end up with more objects than
> are currently there (like sequencing experiements). I can see that Biodata
> maybe not the best name, however specimen doesn't capture it either.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ga4gh/schemas/pull/606/files/7c8111f417bc221e6ce516b4164b5df1ff9838dd#r59670763>
>